wiki/Beatmap/Circle_size/ja.md

```diff
@@ -0,0 +1,38 @@
+---
+tags:
+  - CS
+  - key count
+---
+
+# サークルサイズ
+
+円のサイズに関する基準は[ランキング基準](/wiki/Ranking_Criteria)を参照してください。
```

Comment:
You seems to forget to italicize this line.

```
*円のサイズに関する基準は[ランキング基準](/wiki/Ranking_Criteria)を参照してください。*
```

Reply:
Italics dont apply to Japanese, but do I still have to Italicize it?

Reply:
Oh, I didn't know that. then, It's not necessary to change probably.

Reply:
yeah no need to